src/js/angular/import/templates/import-resource-tree.html

```diff
@@ -149,14 +149,23 @@
                             gdb-tooltip="{{'import.reset.status' | translate}}">
                         <span class="icon-close"></span>
                     </button>
-                    <button class="btn btn-link btn-sm secondary import-resource-action-remove"
+                    <button ng-if="!onDelete" class="btn btn-link btn-sm secondary import-resource-action-remove"
                             ng-click="onDelete({resource: resource})">
                         <em class="icon-trash"></em>
                     </button>
                     <button class="btn btn-primary btn-sm import-resource-action-import"
+                            ng-if="resource.isFile() && resource.importResource.status !== ImportResourceStatus.IMPORTING && resource.importResource.status !== ImportResourceStatus.UPLOADING && resource.importResource.status !== ImportResourceStatus.PENDING && resource.importResource.status !== ImportResourceStatus.INTERRUPTING"
```

Comment:
I wonder if we can precalculate these after each load of the list. Although the status can't be changed in other means. The idea is to have a property `isImportable` in the resource model for example wich to be precalculated and just refered here.